server/gql/models/addresses.js

```diff
@@ -5,20 +5,25 @@ import { supplierQueries } from './suppliers';
 import { timestamps } from './timestamps';
 import db from '@database/models';
 import { storeQueries } from '@gql/models/stores';
-import { totalConnectionFields } from '@utils/index';
+import { totalConnectionFields, getQueryFields, REQUIRED_ARGS } from '@utils/index';
+import { TYPE_ATTRIBUTES } from '@utils/constants';
 
 const { nodeInterface } = getNode();
 export const addressFields = {
   id: { type: GraphQLNonNull(GraphQLID) },
-  address1: { type: GraphQLString },
-  address2: { type: GraphQLString },
-  city: { type: GraphQLString },
-  country: { type: GraphQLString },
+  address1: { type: GraphQLString, ...REQUIRED_ARGS, isUpdateRequired: false },
```

Comment:
```
isUpdateRequired: false
```

if its not required we don't need to explicitly add `isUpdateRequired: false`,  since undefined will be evaluated as false too

Reply:
...REQUIRED_ARGS has isUpdateRequired: true so i changed it